src/views/Auth/Login.jsx

```diff
@@ -32,18 +37,24 @@ export default function Login() {
     <>
       <h3>You must log in to view the page at {from.pathname}</h3>
       <form onSubmit={handleLogin} className={styles.loginForm}>
-        <label>Email</label>
-        <input
-          id="email"
-          name="email"
-          type="email"
-        />{' '}
-        <label>Password</label>
-        <input
-          id="password"
-          name="password"
-          type="password"
-        />
+        <label>
+          Email
+          <input
+            id="email"
+            name="email"
+            type="email"
```

Comment:
We should probably set the value of these inputs to the related state so React has full control of this input